uber/static/theme/prereg.css

```diff
@@ -66,8 +66,8 @@ body {
 }
 
 .prereg-price-notice {
-    color: red;
```

Comment:
This is an unrelated fix; I found the price bump warnings to be a little too unreadable. :)